functions/fishtape.fish

```diff
@@ -65,13 +65,19 @@ function fishtape --description "Test scripts, functions, and plugins in Fish"
                         string replace --filter --regex -- "\s+called on line (\d+) of file (.+)" '$2:$1' |
                         read --local at
 
+                    if test $argv[1] = "!"
+                        set operator "! "
+                        set negation "not "
+                        set --erase argv[1]
+                    end
```

Comment:
Would it work if we move this code to the top of the function instead?

Reply:
Love it! Can do. As always, I really appreciate your amazing work on fish utilities and responsiveness to the community.

Reply:
Oops, I was wrong - this code has to exist in the else part, because it relies on the if part to have all of $argv in place. Once I erase the ! in argv[1], this code won't work unless I complicate the operator, expected, actual logic further. Any recommendations?

https://github.com/jorgebucaran/fishtape/blob/5262b49cd953bf3abc6c147d295f1c461386706c/functions/fishtape.fish#L57-L72

Reply:
True. It's okay. 👍